### PR TITLE
Exclude AC charts

### PR DESCRIPTION
**Pull request by statementreply**

### CODE REVIEW

File: updatenewlive.py [hunk 1]
```diff
@@ -43,6 +43,9 @@
 		livesetting = jpdbconn.execute('SELECT * FROM live_setting_m WHERE live_track_id = '+str(jptmp[0])+';')
 		livetmp = livesetting.fetchone()
 		while livetmp:
+			# Exclude AC charts
+			if livetmp[0] >= 10000:
```
Comment (ben1222) [inline updatenewlive.py:47]: Exclude AC here is not enough, the weight will still be wrong. (the `updateweight.py` in this branch seems not the version that llhelper is currently using...)
And maybe better to use `'SELECT ac_flag FROM special_live_m WHERE live_setting_id = ' + str(livetmp[0]) + ';'` to determine if it is AC

Comment (statementreply) [reply updatenewlive.py:47]: Some "off-shelf" charts doesn't exist in either `normal_live_m` or `special_live_m`. We can't determine whether they are AC from these tables.

Comment (ben1222) [reply updatenewlive.py:47]: Could you give an example? I didn't find "off-shelf" live... at least on the db I received...
You may want to also check the `event_battle_live_m`, `event_challenge_live_m`, `event_festival_live_m`, `event_marathon_live_m`, `event_quest_live_m`, `event_team_duty_live_m`, etc., for the given live setting id.

File: updatenewlive.py [hunk 2]
```diff
@@ -43,6 +43,9 @@
...
+				continue
```
Comment (ben1222) [inline updatenewlive.py:48]: Noticed just now, the `continue` here will result in infinite loop.
Need `livetmp = livesetting.fetchone()` before `continue`

**Pull request merged**